src/mint/types/responses.ts

```diff
@@ -108,9 +108,9 @@ export type GetInfoResponse = {
 };
 
 /**
- * Response from the mint after requesting a melt quote.
+ * Base melt quote response - all melt quotes have these fields (NUT-05)
  */
-export type PartialMeltQuoteResponse = {
+export type NUT05MeltQuoteResponse = {
```

Comment:
Review note: This was an interim name - in #414 it is moved and renamed to `MeltQuoteBaseResponse`

Reply:
MeltQuoteBaseResponse *is* the payload. MeltQuoteBaseRequest is the request - so yes, both were named using same convention.